telemetry/definitions/commonDefinitions.json

```
@@ -22,6 +22,11 @@
             "allowedValues": ["Succeeded", "Failed", "Cancelled"],
             "description": "The result of the operation"
         },
+        {
+            "name": "kind",
+            "allowedValues": ["Viewer", "Default"],
+            "description": "The kind of call made"
```

Our current use-cases for this type is that we want to distinguish the IDE location of `s3_download*` and `s3_upload*` actions. Probably this can just be a generic "UI thing" type. Could be useful for other metrics besides S3. "component" seems like a good name.

```suggestion
            "name": "component",
            "allowedValues": ["editor", "viewer", "filesystem"],
```